ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/DataColumnSidecarCustodyImpl.java

```diff
@@ -103,6 +108,9 @@ public DataColumnSidecarCustodyImpl(
     this.nodeId = nodeId;
     this.totalCustodySubnetCount = totalCustodySubnetCount;
     this.eip7594StartEpoch = spec.getForkSchedule().getFork(SpecMilestone.EIP7594).getEpoch();
+    this.knownSavedIdentifiers =
```

Comment:
May be would better move this optimization down to either  
- `DataColumnSidecarDbImpl`
- `CombinedChainDataClient`
- Or make a dedicated `DataColumnSidecarDb` wrapper to cache this?

Reply:
https://github.com/Nashatyrev/teku/issues/122